Update of Pasteurellosis (#60)
manolinaqua/fish-disease-library: Fish Disease Library/Bacterial Diseases/Pasteurellosis.md

Reviewer: Stylistic note: To keep the text consistent, the first word after each colon in the bullet points should start with a capital letter (e.g., ‘Lethargy: Reduced activity…’, ‘Loss of appetite: Decreased feeding". How the text is now, it switches between capital and lowercase.

## Overview

### What is Pasteurellosis?

> Pasteurellosis is a bacterial disease of significant concern affecting farmed Atlantic salmon (_Salmo salar_) and lumpfish (_Cyclopterus lumpus_) in Norway and Scotland. It is caused by bacteria of the family Pasteurellaceae, genus Phocoenobacter, particularly *Phocoenobacter* *atlanticus* subspecies *atlanticus* and subspecies *cyclopteri*, as well as *Phocoenobacter* *skyensis*. The disease leads to high mortality rates and economic losses in both host species. Pasteurellosis in Atlantic salmon was first identified in Norway in 1989 and has since 2018 become increasingly prevalent, with approximately 50 diagnosed cases registered annually in farms in south-western Norway from 2020 to 2022, suggesting endemic establishment [1](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1236290/full). In lumpfish, the disease was first confirmed in 2012 during their peak introduction as cleaner fish in salmon aquaculture as biological control of the salmon louse L. salmonis [2](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12366?casa_token=B_EZwK3lfp4AAAAA%3ACmW4N_RL8PDRkw8zBtjFzgTdBFtIIucaDXJbJJNvHj1GFoDoygn_gl5Df524Z-npFrWSnFputirfyA). The disease has since then become recurring and severe, leading to up to100% mortality in lumpfish [3](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg). *Phocoenobacter skyensis* is linked to pasteurellosis in Scottish farmed Atlantic salmon [4](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/00207713-52-3-699), with one isolated outbreak in Norwegian farmed salmon in 2020 [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway).

Reviewer: ```suggestion
Pasteurellosis is a bacterial disease of significant concern affecting farmed Atlantic salmon (_Salmo salar_) and lumpfish (_Cyclopterus lumpus_) in Norway and Scotland. It is caused by bacteria of the family Pasteurellaceae, genus Phocoenobacter, particularly *Phocoenobacter* *atlanticus* subspecies *atlanticus* and subspecies *cyclopteri*, as well as *Phocoenobacter* *skyensis*. The disease leads to high mortality rates and economic losses in both host species. Pasteurellosis in Atlantic salmon was first identified in Norway in 1989 and has since 2018 become increasingly prevalent, with approximately 50 diagnosed cases registered annually in farms in south-western Norway from 2020 to 2022, suggesting endemic establishment [1](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1236290/full). In lumpfish, the disease was first confirmed in 2012 during their peak introduction as cleaner fish in salmon aquaculture as biological control of the salmon louse (_Lepeophtheirus salmonis_) [2](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12366?casa_token=B_EZwK3lfp4AAAAA%3ACmW4N_RL8PDRkw8zBtjFzgTdBFtIIucaDXJbJJNvHj1GFoDoygn_gl5Df524Z-npFrWSnFputirfyA). The disease has since then become recurring and severe, leading to up to100% mortality in lumpfish [3](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg). *Phocoenobacter skyensis* is linked to pasteurellosis in Scottish farmed Atlantic salmon [4](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/00207713-52-3-699), with one isolated outbreak in Norwegian farmed salmon in 2020 [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway).
```

Reviewer: Reccomend to write out Lepeophtheirus salmonis in full at first mention instead of using “L. salmonis.” (look at suggestion)

The term pasteurellosis has previously been wrongly associated with disease caused by *Photobacterium* *damselae* subsp. *piscicida* (family Vibrionaceae) (formerly incorrectly classified as *Pasteurella* *piscicida*) [1](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1236290/full). Correctly used, pasteurellosis is defined as the disease caused by *bona fide* species of the Pasteurellaceae family.

## Symptoms of Pasteurellosis

### Common Symptoms

Atlantic Salmon: 

- **Physical Signs:**
    - **Septicemia:** Systemic bacterial infection affecting multiple organs. [6](https://www.vetinst.no/en/diseases/pasteurellosis-in-fish).
    - **Hemorrhage and inflammation:** ulcers in skeletal and cardiac muscle, hemorrhaging in internal organs, and inflammation around the pericardium, abdominal wall, pseudobranchs, and the base of the pectoral fins [6](https://www.vetinst.no/en/diseases/pasteurellosis-in-fish).

>     - **Exophthalmia:**  severe inflammation of the eye and eye socket (varracalbmi) in some cases [6](https://www.vetinst.no/en/diseases/pasteurellosis-in-fish).

Reviewer: ```suggestion
    - **Exophthalmia:**  severe inflammation of the eye and eye socket in some cases [6](https://www.vetinst.no/en/diseases/pasteurellosis-in-fish).
```

Reviewer: Inflammation of the eye and eye socket gave the disease the Norwegian name “Varraccalbmi” (Sámi for “blood eye”) - but this could perhaps be included in the overview section. In suggestions “Varraccalbmi” is removed.

    - **Histopathology**: abundant inflammatory cells, tissue necrosis, and short rod-shaped bacteria in affected organs [6](https://www.vetinst.no/en/diseases/pasteurellosis-in-fish).

- **Behavioral Changes:**
    - **Lethargy:** Reduced activity and sluggish swimming  [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway).
    - **Loss of Appetite:** Decreased feeding behavior [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway).
    - **Abnormal Swimming Patterns:** Fish may swim sluggishly and in random patterns [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway).

**Progression of Symptoms**

- **Early Stages:** Subtle signs such as reduced feed intake and slight lethargy.

> - **Advanced Stages:** disease can become clinically obvious late in the production cycle and may end in severe pathology or harvest decisions because of poor welfare [6](https://www.vetinst.no/en/diseases/pasteurellosis-in-fish), [7](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.006729#tab2).

Reviewer: ```suggestion
- **Advanced Stages:** disease can become clinically obvious late in the production cycle and may end in severe pathology or early harvest decisions because of poor welfare [6](https://www.vetinst.no/en/diseases/pasteurellosis-in-fish), [7](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.006729#tab2).
```

> - **Impact on Fish Health:** Pasteurellosis severely compromises overall health and often leads to significant mortality [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway), [8](https://onlinelibrary.wiley.com/doi/full/10.1111/jfd.13381).

Reviewer: This report: https://www.fhf.no/prosjekter/prosjektbasen/901680/ , highlights this: "The clinical presentation varies, and the direct mortality resulting from an outbreak ranges from low to moderate". Consider to include it in this section perhaps (or something from another source specifically about mortality).

Lumpfish: 

- **Physical Signs:**
    - **Septicemia:** Systemic bacterial infection affecting multiple organs [3](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg),  [8](https://onlinelibrary.wiley.com/doi/full/10.1111/jfd.13381), [9](https://www.sciencedirect.com/science/article/pii/S1050464819310587?casa_token=uYme_FZ-c8YAAAAA:Uj2PMDz9OKLezgXQrOAml6ZwaAtryDO02cCfvtfYFuHFsvoXF1dCU2LfkQoG1mgKlqGkmxve).

>     - **Hemorrhages:** Reddened areas around the jaw and base of fins [3](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg),  [8](https://onlinelibrary.wiley.com/doi/full/10.1111/jfd.13381), [9](https://www.sciencedirect.com/science/article/pii/S1050464819310587?casa_token=uYme_FZ-c8YAAAAA:Uj2PMDz9OKLezgXQrOAml6ZwaAtryDO02cCfvtfYFuHFsvoXF1dCU2LfkQoG1mgKlqGkmxve).

Reviewer: ```suggestion
    - **Hemorrhages:** Reddened areas around the jaw, base of fins and gills [3](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg),  [8](https://onlinelibrary.wiley.com/doi/full/10.1111/jfd.13381), [9](https://www.sciencedirect.com/science/article/pii/S1050464819310587?casa_token=uYme_FZ-c8YAAAAA:Uj2PMDz9OKLezgXQrOAml6ZwaAtryDO02cCfvtfYFuHFsvoXF1dCU2LfkQoG1mgKlqGkmxve).
```

- **Impact on Fish Health:** up to 100% mortality observed in infected lumpfish, with survivors potentially acting as asymptomatic carriers of the bacteria that can lead to re-emergence of disease if triggered by a stress event [3](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg).  Compared with salmon, lumpfish seem highly susceptible and may develop marked pathology even when salmon exposed to similar isolates show few or no clinical signs [8](https://onlinelibrary.wiley.com/doi/full/10.1111/jfd.13381).

### Causes of Pasteurellosis

**Etiology**

- **Causative Agent:** Bacteria in the genus *Phocoenobacter*, namely the species *skyensis*, and *atlanticus* which includes the subspecies *atlanticus* and *cyclopteri* [1](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1236290/full), [3](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg).

- **Transmission Methods:**

>     - **Horizontal transmission:** Typically transmitted between cages within localities, with higher risk of transmission between localities through the use of well boats [10]([Strand et al., 2026](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0347930). Under laboratory conditions, transmission of disease from lumpfish to Atlantic salmon in a cohabitation environment was unsuccessful, however lumpfish are highly susceptible to disease caused by both subspecies [8](https://onlinelibrary.wiley.com/doi/full/10.1111/jfd.13381).

Reviewer: I would be more careful with this statement of horizontal transmission as the papers don't confirm this and Veterinærinstituttets page says: "Routes of infection and survival in the environment are not known, but outbreaks in nearby locations may indicate horizontal transmission." from https://www.vetinst.no/sykdom-og-agens/pasteurellose-hos-fisk. I would suggest to include that instead.

- **Farm Management Practices:** Stress conditions, such as delousing treatments, can increase susceptibility to pasteurellosis. Outbreaks often reported 2–3 weeks after handling such as lice treatment [8](https://onlinelibrary.wiley.com/doi/full/10.1111/jfd.13381), [10]([Strand et al., 2026](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0347930), [11](https://onlinelibrary.wiley.com/doi/full/10.1111/jfd.14085).
- Strong statistical association between **thermal** (OR≈2.5) and **mechanical** (OR≈1.8) delousing and Pasteurella detection; freshwater delousing not associated with increased risk [11](https://onlinelibrary.wiley.com/doi/full/10.1111/jfd.14085).
- Increased salmon weight and previous farm‑cycle Pasteurella diagnosis increase odds of new detection [11](https://onlinelibrary.wiley.com/doi/full/10.1111/jfd.14085).

### Diagnosis

**Diagnostic Methods**

- **Clinical Examination:** Observation of physical and behavioral symptoms [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway), [6](https://www.vetinst.no/en/diseases/pasteurellosis-in-fish).

- **Laboratory Tests:**

>     - **Bacterial Culture:** Isolation and identification of Phocoenobacter atlanticus on blood agar with 2% NaCl. Grows as small grey colonies within 2-4 days, with slight alpha hemolysis. 

Reviewer: Add a reference here for this statement (seems it was removed).

    - **MALDI-TOF MS:** Mass spectrometry for rapid identification [6](https://www.vetinst.no/en/diseases/pasteurellosis-in-fish).
    - **Whole Genome Sequencing:** For precise identification and discrimination of Phocoenobacter atlanticus isolates [1](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1236290/full), [7](https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.006729#tab2).

**Differential Diagnosis**

> - **Distinguishing pasteurellosis from other diseases:** It is crucial to differentiate pasteurellosis from other bacterial diseases with similar symptoms, such as atypical furunculosis (in lumpfish) and typical furunculosis (in Atlantic salmon). Biochemical and molecular testing of the bacteria can be used to distinguish between the two diseases.

Reviewer: Add a reference here as well for this statement (seems it was removed).

Author: The reference was removed because there is no mention of differential diagnosis in this article. The practice is established in diagnostics, and one we teach fish health students here at UiB. That being said, I have not found any scientific articles mentioning this (presumably because it is taken for granted). GenAI is more miss than hit on this disease, so I am limiting its use to collect information.

### Prophylaxis, treatment and control

**Vaccination- lumpfish:**

> - Experimental monovalent vaccines against pasteurellosis induce high specific antibody levels but only **partial protection**, with high mortality still observed and no reduction in bacterial load in survivors [3](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg).

Reviewer: Wrong citation used here. The right one is this paper: https://www.sciencedirect.com/science/article/abs/pii/S1050464819310587

> - Bath challenge and immunology work suggest humoral antibodies alone are insufficient; other immune components are likely important [3](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg).

Reviewer: Wrong citation used here as well. The right one is this paper: https://www.sciencedirect.com/science/article/abs/pii/S1050464819310587

**Antibacterial therapy- lumpfish:**

- In experimental infections, medicated feed with **florfenicol 20 mg/kg/day** gave good early effect against pasteurellosis, improving survival, while oxolinic acid and flumequine were not effective; protection waned later in the trial [15](https://onlinelibrary.wiley.com/doi/10.1111/jfd.13545).
- Effective protocols are important to reduce mortality and resistance risk; this is the first protocol‑oriented study in lumpfish [15](https://onlinelibrary.wiley.com/doi/10.1111/jfd.13545).

> - For hatcheries, antibiotics are used but no approved standardized protocols exist; treatment in sea cages is difficult because of cohabiting salmon [3](https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg).

Reviewer: ```suggestion
- For hatcheries, antibiotics are used but no approved standardized protocols exist; treatment in sea cages is difficult because of cohabiting salmon and the fact that oral administration via medicated feed is often ineffective because dying fish typically stop eating (https://onlinelibrary.wiley.com/doi/abs/10.1111/jfd.12905?casa_token=Adn8S3CfvnsAAAAA:ihryvVUH-Bm88XthU5hHD9xr3BC8EHh7qqPZNG6PHBU4nlVzwG1oijptimWgvllXKnbKFCDZdse3Jg).
```

Reviewer: Suggested to add that oral administration via medicated feed is often ineffective anyways because dying fish typically stop eating

    - **Scotland:** Outbreaks typically associated with *Phocoenobacter skyensis* serotype O2 [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway).

- **Response Strategies and Outcomes:**
    - **Norway:** Increased focus on rapid diagnosis and identification of Pasteurella strains [1](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1236290/full), [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway).
    - **Norway**: The Food Safety Authority is considering listing pasteurellosis in Atlantic salmon a notifiable disease [17](https://hoering.mattilsynet.no/hoering/3823).
    - **Scotland:** Development of serotyping methods for *Ph. skyensis* [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway).

> - **Lessons Learned:** The importance of routine bacteriology, rapid diagnosis, and proper biosecurity measures in controlling pasteurellosis outbreaks [1](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1236290/full), [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway), [16](https://www.tekna.no/fag-og-nettverk/miljo-og-biovitenskap/tekna-havbruk/retningslinjer-for-hygienekontroll-av-fartoy-innen-akvakultur/#forord).

Reviewer: ```suggestion
- **Lessons Learned:** The importance of routine bacteriology, rapid diagnosis, proper biosecurity measures and avoiding stress in controlling pasteurellosis outbreaks [1](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1236290/full), [5](https://eafpbulletin.scholasticahq.com/article/31534-pasteurella-skyensis-in-atlantic-salmon-salmo-salar-l-in-western-norway), [16](https://www.tekna.no/fag-og-nettverk/miljo-og-biovitenskap/tekna-havbruk/retningslinjer-for-hygienekontroll-av-fartoy-innen-akvakultur/#forord).
```

Reviewer: Suggestions to add "avoiding stress"

## Data Insights
### Disease Impact by Country
#### Norway

- **Pasteurellosis Incidence in Norway:**
    - Pasteurellosis was first diagnosed in Atlantic salmon in Norway in 1989 [18](https://onlinelibrary.wiley.com/doi/abs/10.1046/j.1365-2761.2000.00209.x?casa_token=0rEa1fVf-nsAAAAA%3Adwz2-H5OP5eIZ4Hy-lJZptZC-iDweg0kafETyhGp9FsfsXav9a03Rw2g4q7N0BK7SqLEc8VKR8igrQ).
    - It has been a rapidly increasing problem in Atlantic salmon in Western Norway since 2018

>     - Approximately 50 diagnosed cases were registered annually in farms in south-western Norway between 2020 and 2022 [1](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1236290/full).

Reviewer: All of these bullet points have been mentioned before - perhaps remove them from other sections so it's not repeated.

Author: Agreed about repetition, but I think it presents a good summary as it is.

10. Strand, D. A., Wiik-Nielsen, J., Mohammad, S., Nilsen, H., Langhelle, B., Holmeset, M., Patel, S., Log Persson, H., & Colquhoun, D. (2026). Environmental DNA screening of *Phocoenobacter* *atlanticus* subsp. *atlanticus* in Atlantic salmon aquaculture. _PLOS ONE_, _21_(4), e0347930. [https://doi.org/10.1371/JOURNAL.PONE.0347930](https://doi.org/10.1371/JOURNAL.PONE.0347930)
11. Stige, L. C., Colquhoun, D. J., and Oliveira, V. H. S. (2025). Associations between delousing practices and pasteurellosis in farmed atlantic salmon. J. Fish Dis. 48:e14085. [https://doi.org/10.1111/jfd.14085](https://doi.org/10.1111/jfd.14085)
12. Ellul, R., Kalatzis, P. G., Frantzen, C., Haugland, G. T., Gulla, S., Colquhoun, D. J., Middelboe, M., Wergeland, H. I., & Rønneseth, A. (2021). Genomic Analysis of _Pasteurella atlantica_ Provides Insight on Its Virulence Factors and Phylogeny and Highlights the Potential of Reverse Vaccinology in Aquaculture. _Microorganisms 2021, Vol. 9, Page 1215_, _9_(6), 1215. [https://doi.org/10.3390/MICROORGANISMS9061215](https://doi.org/10.3390/MICROORGANISMS9061215)

> 13. Moldal T, Wiik-Nielsen J, Oliveira VHS, Svendsen JC og Sommerset I. Fiskehelserapporten 2024, Veterinærinstituttets rapportserie nr. 1a/2025, utgitt av Veterinærinstituttet 2025. [https://www.vetinst.no/rapporter-og-publikasjoner/rapporter/2025/fiskehelserapporten-2024//attachment/inline/a9141dc4-f516-47be-9261-5efe946bc579:54f6073452f4b7923f3355f3afc8e11308075a28/Fiskehelserapporten%202024_Utgave%202.pdf](https://www.vetinst.no/rapporter-og-publikasjoner/rapporter/2025/fiskehelserapporten-2024/_/attachment/inline/a9141dc4-f516-47be-9261-5efe946bc579:54f6073452f4b7923f3355f3afc8e11308075a28/Fiskehelserapporten%202024_Utgave%202.pdf)

Reviewer: This is incorrect- the new link added does not work (while the old one is correct), so this suggestion should be discarded.

> 14. Moldal T, Wiik-Nielsen J, Oliveira VHS, Svendsen JC og ­Sommerset I. Fiskehelserapporten 2025, Veterinærinstituttets rapportserie nr. 5a/2026, utgitt av Veterinærinstituttet 2026. [https://www.vetinst.no/rapporter-og-publikasjoner/rapporter/2026/fiskehelserapporten-2025//attachment/inline/363a7595-e0c3-4ee8-87e6-71ba808b4288:b3c8e9bf1ee7902bdc1ab93334e5219666ceabbd/Fiskehelserapporten%202025.pdf](https://www.vetinst.no/rapporter-og-publikasjoner/rapporter/2026/fiskehelserapporten-2025/_/attachment/inline/363a7595-e0c3-4ee8-87e6-71ba808b4288:b3c8e9bf1ee7902bdc1ab93334e5219666ceabbd/Fiskehelserapporten%202025.pdf)

Reviewer: Same as last comment: 
This is incorrect- the new link added does not work (while the old one is correct), so this suggestion should be discarded.
